src/effects/native/flangereffect.cpp

```diff
@@ -4,9 +4,9 @@
 
 #include "util/math.h"
 
-const unsigned int kMaxDelay = 5000;
-const unsigned int kLfoAmplitude = 240;
-const unsigned int kAverageDelayLength = 250;
+const unsigned int kMaxDelay = 441000000;
```

Comment:
I thought they were previously all in units of samples, but I could be wrong. @JosepMaJAZ or @rryan could you clarify?